ingress/controllers/nginx/nginx.tmpl

```diff
@@ -16,7 +16,13 @@ events {
 }
 
 http {
-    {{ if $cfg.enableVtsStatus}}vhost_traffic_status_zone shared:vhost_traffic_status:{{ $cfg.vtsStatusZoneSize }};{{ end }}
+    geoip_country       /etc/nginx/GeoIP.dat;
```

Comment:
please update the REAMDE with an example of who this is useful, or some eager newbie will think it's a dead module and optimize it out of the build for sure